zerver/lib/bugdown/__init__.py

```diff
@@ -693,7 +693,12 @@ def make_emoji(emoji_name, src, display_string):
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
-    elt.set("title", display_string)
+    if display_string[0] == ':':
+        # removes the colons at the ends and replaces space with underscore
+        elt.set("title", display_string[1:-1].replace("_", " "))
+    else:
+        # for unicode emojis
+        elt.set("title", display_string)
```

Comment:
The caller could either pass both in or just pass in one version and modify it; it seems like there's no need to call it sometimes with one sort of input string value and sometimes with another.  The callers could agree that they'll always pass with the colons or without, for example.